src/confd/yang/infix-system@2023-10-19.yang

```diff
@@ -76,8 +84,13 @@ module infix-system {
     }
   }
 
-  deviation "/sys:system/sys:clock/sys:timezone/sys:timezone-utc-offset" {
-    deviate not-supported;
+  deviation "/sys:system/sys:clock/sys:timezone/sys:timezone-utc-offset/sys:timezone-utc-offset" {
+    deviate replace {
+      type int16 {
+        range "-12 .. 14";
+      }
+      units "hours";
+    }
   }
```

Comment:
Like we discussed AFK with @jovatn, please add a description for you deviation describing what it does.  This can be seen by a NETCONF or CLI user and describes the limitation of the setting in Infix based products.